css/stylesheet.css

```diff
@@ -88,6 +96,7 @@ nav a {
 nav ul {
   padding: 40px;
   position: relative;
+  float: right;
```

Comment:
Nice switch putting the float on the `<ul>` instead of each `<li>`. This will preserve the order of the nav items.